include/HypreLinearSystem.h

```diff
@@ -20,6 +20,9 @@
 #endif // HYPRE_LINEAR_SYSTEM_DEBUG
 #undef HYPRE_LINEAR_SYSTEM_DEBUG
 
+#include <iostream>
+#include <fstream>
```

Comment:
Do we need these headers here? Move them to `C++`?

Reply:
I was using these for some IO defined by dumpHypreMatrixStats. What do you suggest?

Reply:
Can you show me the specific code in this file? I am assuming the functions/classes from these headers are used in the C++ file?

Reply:
The function that uses this HypreLinearSystem.C::dumpRowNNZStats(). Line 1577

Reply:
https://github.com/PaulMullowney/nalu-wind/blob/hypre_rows_counts/src/HypreLinearSystem.C#L1577

Reply:
Thanks. Any reason then that these headers are not in `HypreLinearSystem.C`

Reply:
Fixed in the most recent commit.